src/hssm/plotting/posterior_predictive.py

```diff
@@ -0,0 +1,199 @@
+        The InferenceData object with posterior samples. If not provided, will use the
+        traces object stored inside the model. If posterior predictive samples are not
+        present in this object, will generate posterior predictive samples using the
+        this InferenceData object and the original data.
+    data : optional
+        The original data. If not provided, will use the data object stored inside the
+        model. If posterior predictive samples are not present in this object, will
+        generate posterior predictive samples using the idata object and the original
+        data in this DataFrame.
+    n_samples : optional
+        When idata is provided, the number or proportion of posterior predictive samples
+        randomly drawn to be used from each chain for plotting. When idata is not
+        provided, the number or proportion of posterior samples to be used to generate
+        posterior predictive samples. The number or proportion are defined as follows:
+
+        - When an integer >= 1, the number of samples to be extracted from the draw
+          dimension.
+        - When a float between 0 and 1, the proportion of samples to be extracted from
+          the draw dimension.
+        - When None, all samples are extracted.
+    row : optional
+        Variables that define subsets of the data, which will be drawn on the row
+        dimension of the facets in the grid. When both row and col are None, one single
+        plot will be produced, by default None.
+    col : optional
+        Variables that define subsets of the data, which will be drawn on the column
+        dimension of the facets in the grid. When both row and col are None, one single
+        plot will be produced, by default None.
+    col_wrap : optional
+        “Wrap” the column variable at this width, so that the column facets span
+        multiple rows. Incompatible with a row facet., by default None.
+    palette : optional
+        Colors to use for the different levels of the hue variable. Should be something
+        that can be interpreted by color_palette(), or a dictionary mapping hue levels
+        to matplotlib colors., by default None.
+    binwidth : optional
+        A number or a pair of numbers. Width of each bin. There are two other binning
+        options: `bins` and `binrange`. Please see the documentation for the
+        [`sns.histplot` function]
+        (https://seaborn.pydata.org/generated/seaborn.histplot.html) for how to use
+        them. in combination to control the bins. This argument overrides bins but can
+        be used with `binrange.`
+    grid_kwargs : optional
+        Additional keyword arguments are passed to the [`sns.FacetGrid` constructor]
+        (https://seaborn.pydata.org/generated/seaborn.FacetGrid.html#seaborn.FacetGrid.__init__)
+        when any of row or col is provided. When producing a single plot, these
+        arguments are ignored.
+    kwargs : optional
```

Comment:
histplot also allows setting to `kde` right?
will this override (or correspondingly ignore) the setting of binwidth?

Reply:
Right now we are not allowing `kde`, but I can add this option. Then it will work however seaborn decides

Reply:
yeah that would make sense, because some people I think are biased towards liking smooth plots :D